linera-rpc/src/conversions.rs

```diff
@@ -252,7 +252,7 @@ impl TryFrom<HandleCertificateRequest> for grpc::Certificate {
             chain_id: Some(request.certificate.value().chain_id().into()),
             value: bincode::serialize(&request.certificate.value)?,
             round: request.certificate.round.0,
-            signatures: bincode::serialize(&request.certificate.signatures)?,
+            signatures: bincode::serialize(&request.certificate.signatures())?,
```

Comment:
```suggestion
            signatures: bincode::serialize(request.certificate.signatures())?,
```

Reply:
Ah yeah, don't need the `&` anymore, good catch